bin/localeapp

```diff
@@ -128,9 +128,15 @@ module LocaleappGLIWrapper
   end
 
   desc "Pulls all translations from localeapp.com"
+  arg_name "[target_dir]"
   command :pull do |c|
     c.action do |global_options, options, args|
-      Localeapp::CLI::Pull.new(global_options).execute
+      target_dir = args.shift
```

Comment:
I have no idea. I tried to follow the conventions used throughout this gem, but I'm unfamiliar with GLI personally.